### PR TITLE
columns will be assembled into rows

### LINKED ISSUE #4: merge single column data
Merge the single-column data of influxdb into the whole row according to the time as much as possible

### PR DESCRIPTION
**Pull request by huangqz71**
feat: when migrating data, columns are assembled into rows before being written

fix #4

### CODE REVIEW

File: src/dataMigrate_test.go [hunk 1]
```diff
@@ -10,250 +10,257 @@ copyright 2023 Huawei Cloud Computing Technologies Co., Ltd.
...
-    "os"
-    "sort"
-    "strconv"
-    "strings"
-    "testing"
-  
-    "net/http"
-    "net/http/httptest"
-    "github.com/influxdata/influxdb/tsdb/engine/tsm1"
+	"fmt"
```
Comment (xiangyu5632) [inline src/dataMigrate_test.go]: please set the tab with 4 blank and reformat all files

File: src/cursor.go [hunk 2]
```diff
@@ -0,0 +1,261 @@
+package main
```
Comment (xiangyu5632) [inline src/cursor.go:15]: Please add the Apache license statement to the file header. 
by the way, if you reference some other open-source projects' source code,  we need to describe the code come from  and the modified part by ourself (like dataMirgarte.go)，**this new file‘s copyright set to yourself**

Comment (huangqz71) [reply src/cursor.go:15]: Sure, I'll deal with it.

File: src/cursor.go [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    et     int64
+    readTs int64
+    key    []byte
+    // seeks is all the file locations that we need to return during iteration.
+    seeks []*location
+    buf   []tsm1.Value
+    pos   int
+}
+
+func (c *Cursor) init(st int64) error {
```
Comment (shilinlee) [inline src/cursor.go]: `st` is unused variable

**Review by shilinlee — CHANGES_REQUESTED** [review]

**Review by shilinlee — APPROVED** [review]

**Pull request merged**